db/create_analytics_views.sql

```diff
@@ -15,51 +15,192 @@
 
 CREATE OR REPLACE VIEW analytics.access_logs AS
-    SELECT id, client_id, created_at, event_type, updated_at, user_id, user_agent FROM public.access_logs;
+    SELECT id, client_id, created_at, event_type, updated_at, user_id, user_agent
+    FROM public.access_logs;
 
 CREATE OR REPLACE VIEW analytics.active_storage_attachments AS
-    SELECT id, blob_id, created_at, record_id, record_type FROM public.active_storage_attachments;
+    SELECT id, blob_id, created_at, record_id, record_type
+    FROM public.active_storage_attachments;
 
 CREATE OR REPLACE VIEW analytics.active_storage_blobs AS
-    SELECT id, byte_size, checksum, content_type, created_at, key FROM public.active_storage_blobs;
+    SELECT id, byte_size, checksum, content_type, created_at, key
+    FROM public.active_storage_blobs;
 
 CREATE OR REPLACE VIEW analytics.admin_roles AS
-    SELECT id, created_at, updated_at FROM public.admin_roles;
-
--- TODO: anonymized_diy_intake_csv_extracts
--- TODO: anonymized_intake_csv_extracts
--- TODO: client_success_roles
--- TODO: clients
--- TODO: coalition_lead_roles
--- TODO: coalitions
--- TODO: delayed_jobs
--- TODO: dependents
--- TODO: diy_intakes
--- TODO: documents
--- TODO: documents_requests
--- TODO: greeter_coalition_join_records
--- TODO: greeter_organization_join_records
--- TODO: greeter_roles
--- TODO: incoming_emails
--- TODO: incoming_text_messages
--- TODO: intakes
-CREATE OR REPLACE VIEW analytics.intakes as
-    SELECT id from intakes;
--- TODO: notes
--- TODO: organization_lead_roles
--- TODO: outbound_calls
--- TODO: outgoing_emails
--- TODO: outgoing_text_messages
--- TODO: provider_scrapes
--- TODO: signups
--- TODO: site_coordinator_roles
--- TODO: source_parameters
--- TODO: stimulus_triages
--- TODO: system_notes
--- TODO: tax_returns
--- TODO: team_member_roles
--- TODO: users
--- TODO: vita_partner_states
--- TODO: vita_partner_zip_codes
--- TODO: vita_partners
--- TODO: vita_providers
+    SELECT id, created_at, updated_at
+    FROM public.admin_roles;
+
+CREATE OR REPLACE VIEW analytics.anonymized_diy_intake_csv_extracts AS
+    SELECT id, created_at, record_count, run_at, updated_at
+    FROM public.anonymized_diy_intake_csv_extracts;
+
+CREATE OR REPLACE VIEW analytics.anonymized_intake_csv_extracts AS
+    SELECT id, created_at, record_count, run_at, updated_at
+    FROM public.anonymized_intake_csv_extracts;
+
+CREATE OR REPLACE VIEW analytics.client_success_roles AS
+    SELECT id, created_at, updated_at
+    FROM public.client_success_roles;
+
+CREATE OR REPLACE VIEW analytics.clients AS
+    SELECT id, attention_needed_since, created_at, current_sign_in_at, failed_attempts, first_unanswered_incoming_interaction_at, last_incoming_interaction_at, last_internal_or_outgoing_interaction_at, last_sign_in_at, locked_at, login_requested_at, routing_method, sign_in_count, updated_at, vita_partner_id
+    FROM public.clients;
+
+CREATE OR REPLACE VIEW analytics.coalition_lead_roles AS
```

Comment:
wondering if we can do a select.* here, or if you made a conscious choice to never do a select.* with the thinking that at some point we will have sensitive data we dont want to expose.

ALSO, is there a way we can say that we DO send all user data to metabase (non anonymized) and only anonymize client data? I think it would help to expose things like "who are the admins" without us doing a query by ssh-ing into aptible.

Reply:
I did make a conscious decision to never `SELECT *` in the spirit of treating this as a whitelist.

> ALSO, is there a way we can say that we DO send all user data to metabase (non anonymized) and only anonymize client data? I think it would help to expose things like "who are the admins" without us doing a query by ssh-ing into aptible.

I think this is a good idea. It would mean we have to trust everyone with Metabase access to view that information. I propose we revisit this question in about 7 days, just to give product people used to using Metabase and have more qualitative sense of how we want to use it.